include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -691,6 +691,8 @@ FEPointEvaluation<n_components, dim, spacedim, Number>::FEPointEvaluation(
   , update_flags(update_flags)
   , update_flags_mapping(update_default)
 {
+  AssertIndexRange(first_selected_component + n_components, fe.n_components ());
```

Comment:
Shouldn't this be
```suggestion
  AssertIndexRange(first_selected_component + n_components, fe.n_components() + 1);
```
You will also need to make a commit with a valid author name (first name + last name), as this is what the indent test fails about.

Reply:
Yes, you're right. Thanks!